_javascripts/page-loader.js

```diff
@@ -1,114 +1,68 @@
-// the new final link to load
-newLink = "";
-newVersion = "";
-currentVersion = "";
-
-// the fileRequested
-fileRequested = "";
-
-//Array prototype to check if an element is in an array
-Array.prototype.contains = function(obj) {
-  return this.indexOf(obj) > -1;
+let newLink = "";
+let newVersion = "";
+let currentVersion = "";
+let fileRequested = "";
+
+const urlMappings = {
+  "openshift-acs": "https://docs.openshift.com/acs/",
+  "openshift-builds": "https://docs.openshift.com/builds/",
+  "openshift-enterprise": "https://docs.openshift.com/container-platform/",
+  "openshift-gitops": "https://docs.openshift.com/gitops/",
+  "openshift-origin": "https://docs.okd.io/",
+  "openshift-pipelines": "https://docs.openshift.com/pipelines/",
+  "openshift-serverless": "https://docs.openshift.com/serverless/",
```

Comment:
These should probably point to the correct path we use for each distro, we need to updated this for `builds, gitops` and others.

Reply:
@Srivaralakshmi @shipsing please comment on whether these locations are correct or gitops and builds

Reply:
yes, the location for builds is correct.

Reply:
yes, the base url location for gitops is correct.